diff --git a/node_modules/oas/dist/chunk-5KFARTQ3.js b/node_modules/oas/dist/chunk-5KFARTQ3.js
index b2b9f0c..575531b 100644
--- a/node_modules/oas/dist/chunk-5KFARTQ3.js
+++ b/node_modules/oas/dist/chunk-5KFARTQ3.js
@@ -186,9 +186,9 @@ function toJSONSchema(data, opts = {}) {
             // merging an `allOf` but because generally when you're merging two schemas together
             // with an `allOf` you want data in the subsequent schemas to be applied to the first
             // and `description` should be a part of that.
-            description: (obj) => {
-              return obj.slice(-1)[0];
-            },
+            description: undefined,
+
+
             // `merge-json-schema-allof` doesn't support merging enum arrays but since that's a
             // safe and simple operation as enums always contain primitives we can handle it
             // ourselves with a custom resolver.
